static/templates/suggestion.json

```diff
@@ -2,6 +2,10 @@
   {{#suggestions}}{{^first}},{{/first}}
   {
     "value" : "{{value}}",
-    "label" : "{{label}}"
+    "label" : "{{label}}",
+    "kind" : "{{kind}}",
+    {{#path}}
+      "path" : "{{path}}"
+    {{/path}}
```

Comment:
For an empty `path` this will result in an invalid JSON (a comma followed by a closing brace, which confuses the strict parsers).

Reply:
so what should I do about it?....I confirm for empty path my current solution works fine

Reply:
Move the comma

Reply:
thanks I understand what you meant now